index.html

```diff
@@ -166,7 +164,7 @@ <h1 class="text-center">About Us</h1>
                     <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
-                    <form action="">
+                    <form action="noaction.php">
```

Comment:
Use a # instead of `noaction.php` coz if we click on submit it will goto to `noaction.php` which is an invalid file.